kernel/os/src/arch/cortex_m33/os_fault.c

```diff
@@ -33,6 +33,12 @@
 #include "reboot/log_reboot.h"
 #endif
 
+#if MYNEWT_VAL(MEMFAULT_ENABLE)
```

Comment:
Not needed since the callback is now abstracted out

Reply:
Fixed